src/jdataview.js

```diff
@@ -244,6 +240,71 @@ Int64.fromNumber = function (number) {
 	return new Int64(lo, hi);
 };
 
+
+// Precise versions of toString() for Int64/Uint64
+
+Uint64.prototype.toString = function() {
+	// Faster toString() for numbers which can be represented precisely as floats.
+	if (this.hi < pow2(19)) {
+		return Number.prototype.toString.apply(this.valueOf(), arguments);
+	}
+
+	// This converts the numbers to base 10 digit arrays for arbitrary precision toString().
+	// See http://www.danvk.org/hex2dec.html
+
+	// e.g. 1234 -> [4, 3, 2, 1]
+	var numToDigits = function(num) {
+		var digits = num.toString().split('');
+		for (var i = 0; i < digits.length; i++) {
+			digits[i] = +digits[i];
+		}
+		digits.reverse();
+		return digits;
+	};
+
+	// Adds two digit arrays for the given base, returning the result.
+	var add = function(x, y, base) {
```

Comment:
I don't think we would need generic `base` in this case. Can you please inline it?